zds/notification/models.py

```diff
@@ -305,10 +308,35 @@ def get_notification_title(self, content):
         return content.title
 
 
+class PingSubscription(AnswerSubscription, MultipleNotificationsMixin):
+    """
+    Subscription to ping of a user
+    """
+    module = _(u'Ping')
+    objects = SubscriptionManager()
+
+    def __unicode__(self):
+        return _(u'<Abonnement du membre "{0}" aux mentions>').format(self.profile, self.object_id)
+
+    def get_notification_title(self, answer):
+        assert hasattr(answer, 'author')
+        assert hasattr(answer, "get_notification_title")
+
+        return _(u'{0} vous a mentionné sur {1}.').format(answer.author, answer.get_notification_title())
```

Comment:
Les logs travis disent `TypeError: int() argument must be a string or a number, not 'SimpleLazyObject'`, et je pense que si tu vires l'i18n ici ça passe. Mais j'ai pas tiré ta PR en local pour tester, c'est qu'une hypothèse.

Et il vaut pas virer l'i18n si c'est le cas, il faut trouver une meilleure solution.